### PR TITLE
BCF-10034: set system-desc for uplink LLDPs appropriately

### PR DESCRIPTION
**Pull request by wolverineav**
Reviewer: @sarath-kumar 

 - default is set to STATIC, as usual
 - for centos + Fuel - changes to LACP

### CODE REVIEW

File: bosi/lib/node.py [hunk 1]
```diff
@@ -487,83 +487,83 @@ def __str__(self):
...
-            'rabbit_hosts': self.rabbit_hosts,
-            'keystone_auth_url': self.keystone_auth_url,
-            'keystone_auth_user': self.keystone_auth_user,
-            'keystone_password': self.keystone_password,
-            'keystone_auth_tenant': self.keystone_auth_tenant,
-            'time_diff': self.time_diff,
-            'last_log': self.last_log,
-            'rhosp_automate_register': self.rhosp_automate_register,
-            'rhosp_installer_management_interface':
+             'bash_script_path': self.bash_script_path,
```
Comment (wolverineav) [inline bosi/lib/node.py]: apparently there were leading spaces :|

**Comment by sarath-kumar** [comment]
Did we test these changes against RHOSP to see controller nodes come up fine? IOW, how is current RHOSP integration working with existing LACP default?

**Comment by wolverineav** [comment]
> Did we test these changes against RHOSP to see controller nodes come up fine? IOW, how is current RHOSP integration working with existing LACP default?

i'll have to check RHOSP P-only env to check how its still working (with LACP bond mode. i'm assuming LACP is enabled) - but this change won't affect any RHOSP env, since BOSI only does package upgrades for RHOSP.

**Comment by sarath-kumar** [comment]
Sounds fine, but we need to get to the bottom of RHOSP.

**Pull request merged**